triage/views.py

```diff
@@ -169,7 +170,15 @@ def dispatch(self, request, *args, **kwargs):
                 kwargs={'step': TriageWizardFormView.SECTOR}
             )
             return redirect(url)
-        return super().dispatch(request, *args, **kwargs)
+        response = super().dispatch(request, *args, **kwargs)
+        response.set_cookie(
```

Comment:
I set the cookie on the custom page view, rather than on triage complete view.

setting the cookie on triage completed would result in the cookie being invalidated too early: if the user has not been to the triage for a long time their cookie will be invalidated, but if they visit the custom page often then it wont be lost.

Reply:
Maybe we should name the cookie more specifically (like the sso_display_login), as it should only be used on the domestic landing page to check if the triage was completed. A name discouraging from usage anywhere else would be best, something like `DO_NOT_USE_THIS_ANYWHERE_ELSE_THAN_STATIC_LANDING_PAGE_COMPLETED_TRIAGE` ;-)

We also need a test checking that DB is not accessed during rendering of the static pages